app/views/layouts/_navbar.html.haml

```diff
@@ -2,23 +2,29 @@
   .ui.container
     .header.item
       = link_to root_path do
-        = semantic_icon('home')
-        = t('.home')
-    .header.item
-      = link_to companies_path do
-        = semantic_icon('building outline')
-        = t('.search')
-    - unless user_signed_in?
-      .header.item
-        = link_to new_user_session_path do
-          = semantic_icon('sign in')
-          = t('.sign_in')
-      .header.item
-        = link_to new_user_registration_path do
-          = semantic_icon('sign in')
-          = t('.sign_up')
+        = 'E-Conseils'
```

Comment:
Should the title be extracted in a I18n variable to allow easy and consistant change if the name does change ?